R/calibrate.R

```diff
@@ -10,7 +10,7 @@ run_calibrate <- function(model_output, calibration_options, path_results,
     })
   }
 
-  path_results <- normalizePath(path_results, mustWork = TRUE)
+  path_results <- normalizePath(path_results, winslash = "/", mustWork = TRUE)
```

Comment:
I suggest making a utility

```
normalise_path <- function(path) {
  normalizePath(path, winslash = "/", mustWork = TRUE)
}
```

and using that everywhere. Just as much noise in the diff but easier to deal with in future